crypto/asn1/a_strex.c

```diff
@@ -471,6 +471,14 @@ static int do_name_ex(char_io *io_ch, void *arg, const X509_NAME *n,
         indent = 0;
         break;
 
+        case XN_FLAG_SEP_SLASH_PLUS:
```

Comment:
nit: Indent is off by 4 spaces here

The other style errors are in keeping with the style of the file, and I think can be waived.

Reply:
Nice, fixed. Will push shortly.